.github/workflows/ci.yaml

```diff
@@ -119,6 +119,47 @@ jobs:
         shell: zsh {0}
       - run: make test-e2e
         shell: zsh {0}
+  e2e-tests-for-docker-compat:
```

Comment:
This is heavy but we need it to ensure there is no regression when FINCH_DOCKER_COMPAT. Let me know if anyone has better ideas.

Reply:
We probably don't need to test the new flag on every platform since they should all have the same version of nerdctl/dependencies installed on them

Reply:
I feel the same way, seems a bit excessive to test on all combinations.

Reply:
Will go macOS 13 only